lib/prefilter.js

```diff
@@ -0,0 +1,52 @@
+const SierraBib = require('./sierra-models/bib')
+const SierraItem = require('./sierra-models/item')
+const SierraHolding = require('./sierra-models/holding')
+const { client } = require('./elastic-search/client')
+const { uriForRecordIdentifier } = require('./utils/uriForRecordIdentifier')
+
+const prefilterBibs = (bibs) => {
+  return bibs
+    .map(bib => new SierraBib(bib))
+    .filter((bib) => {
+      const suppress = bib.isResearch() &&
+        !bib.getSuppressionWithRationale().suppressed
+
+      if (suppress) {
+        suppressBib(bib)
```

Comment:
Need a way to _not_ issue this DELETE call if we're just building the document for debugging purposes. For example, the `scripts/compare-with-indexed` script invokes `buildEsDocument`, which calls `prefilterBibs`. Perhaps a `options` arg to `buildEsDocument` and `prefilterBibs` that defines a `deleteSuppressedBibs` option, which defaults to `false`. We would set to to `true` when calling `buildEsDocument` from `index.js`..

Reply:
Currently the setup is to use the following check:
```process.env.DISABLE_CIRC_DELETE === 'true'
```
to disable deletion. What if we keep the environment variable approach but change the default behavior to not delete and change the variable name to reflect that? I've made that change but let me know if you don't like it